src/ring_theory/ideal/operations.lean

```diff
@@ -1139,29 +1139,34 @@ end
 
 section quotient_algebra
 
-/-- The `R`-algebra structure on `R/I` -/
-instance {I : ideal R} : algebra R (ideal.quotient I) := (ideal.quotient.mk I).to_algebra
+variables (R) {A : Type*} [comm_ring A] [algebra R A]
 
-/-- The canonical morphism `R →ₐ[R] I.quotient`, for `I` an ideal of `R`, as morphism of
-`R`-algebras. -/
-def quotient.mkₐ (I : ideal R) : R →ₐ[R] I.quotient :=
+/-- The `R`-algebra structure on `A/I` for an `R`-algebra `A` -/
+instance {I : ideal A} : algebra R (ideal.quotient I) :=
+(ring_hom.comp (ideal.quotient.mk I) (algebra_map R A)).to_algebra
```

Comment:
You should also be able to prove something like
```lean
instance {I : ideal A} [is_scalar_tower R S A] : is_scalar_tower R S (ideal.quotient I) :=
```
which would be handy to have.

Reply:
I added 
```lean
instance {I : ideal A} : is_scalar_tower R A (ideal.quotient I)
```
No need to assume some `is_scalar_tower` :)